lib/src/blocs/movies_bloc.dart

```diff
@@ -0,0 +1,26 @@
+import 'dart:async';
+import '../models/item_model.dart';
+import '../resources/movies_repository.dart';
+import 'i_movies_bloc.dart';
+
+class MoviesBloc extends IMoviesBloc {
+  MoviesRepository moviesRepository = MoviesRepository();
+  StreamController<ItemModel> moviesStreamController = StreamController();
+
+  @override
+  void initialize() {}
+
+  @override
+  void dispose() {
+    this.moviesStreamController.close();
```

Comment:
Using "this" is the same as not using it. Remove it of all this file.

Reply:
done